README.md

```diff
@@ -31,8 +31,38 @@ When the reader has completed this Code Pattern, they will understand how to:
 * [JavaScript](https://developer.mozilla.org/bm/docs/Web/JavaScript) - this demo uses no frameworks, just plain vanilla JavaScript.
 * [Apache CouchDB](http://couchdb.apache.org/) - modern, document database hosted on your server or in the cloud.
 
+## Key concepts
+
+This shopping list app is a small single page web application consisting of an HTML file, a couple of CSS files, and a couple of JavaScript files, the PouchDB library but no other JavaScript frameworks (i.e., jQuery, Ember.js, etc.). The web page will allow multiple shopping lists to be created (e.g., Groceries, Clothes, etc.) each with a number of shopping list items associated with them (e.g., Bread, Water, etc.).
+
+So what sets this app apart? Its Offline First architecture. The Offline First approach plans for the most constrained network environment first, enabling a great user experience even while the device is offline or has only an intermittent connection, and providing progressive enhancement as network conditions improve. This design also makes the app incredibly performant (fast!) on the best of networks. 
+
+PouchDB, CouchDB, and Service Worker are the primary tools that turn our simple shopping list app into a high performance, offline-capable Progressive Web App.
+
+**Data stays safe on your device, even while it's offline.**  <br>Persistance of shopping lists and item data entered by the user is achieved using the in-browser database PouchDB. This will allow your data to survive between sessions and when disconnected from the network. (Whether you remember that you need juice while you're on your trusty home WiFi or in the middle of the wilderness, you can still add it your list.)
```

Comment:
Markdown formatting note: You can do a newline by entering two spaces at the end of the previous line and hitting enter.
https://daringfireball.net/projects/markdown/syntax#p